packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -202,9 +223,10 @@ export class FluidContainer extends TypedEventEmitter<IFluidContainerEvents> imp
     }
 
     /**
-     * {@inheritDoc IFluidContainer.attach}
+     * {@inheritDoc IFluidContainer.connect}
```

Comment:
Attach and connect are different.
```suggestion
     * {@inheritDoc IFluidContainer.attach}
```